go.mod

```diff
@@ -13,4 +13,4 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.5.0
 	github.com/spotinst/spotinst-sdk-go v1.174.0
 	golang.org/x/lint v0.0.0-20200302205851-738671d3881b
-)
+)
```

Comment:
fix the change in this file otherwise it will not allow to merge

Reply:
Updated go.mod file with latest SDK version.